graviti/utility/file.py

```diff
@@ -43,6 +56,25 @@ def checksum(self) -> str:
         """
         return self._checksum
 
+    @staticmethod
+    def from_pyarrow(scalar: pa.StructScalar) -> Union["RemoteFile", "File"]:
+        """Create File or RemoteFile instance with pyarrow struct scalar.
+
+        Arguments:
+            scalar: The input pyarrow struct scalar.
+
+        Returns:
+            The File or RemoteFile instance.
+
+        """
+        url: str = scalar["url"].as_py()
+        checksum: str = scalar["checksum"].as_py()
+
+        if url.startswith("file://"):
+            return File._create(url, checksum)  # pylint: disable=protected-access
+
+        return RemoteFile(url, checksum)
+
 
 class RemoteFile(FileBase):
```

Comment:
Does `RemoteFile` need a `to_pyobj` method?

Reply:
Is not necessary for now